src/content/advanced/cluster-management/node-pools-capi/index.md

```diff
@@ -0,0 +1,322 @@
+{{< tab id="cluster-capa-ec2" for-impl="capa_ec2" >}}
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+  nodeSelector:
+    giantswarm.io/machine-pool: a1b2c3-pool0
+```
+
+{{< /tab >}}
+{{< tab id="cluster-capz-azure-vms" for-impl="capz_vms" >}}
+
+```yaml
+A similar example for an Azure cluster:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nginx
+spec:
+  containers:
+  - name: nginx
+    image: nginx
+  nodeSelector:
+    giantswarm.io/machine-deployment: a1b2c3-pool0
+```
+
+{{< /tab >}}
+{{< /tabs >}}
+
+You can assign workloads to node pools in a more indirect way too. This is achieved by using other node attributes which are
+specified via the node pool and which are exposed as node labels.
+
+For example: In a case where you have node pools with one instance type. Using a `nodeSelector` with the label `node.kubernetes.io/instance-type` you can assign workloads to matching nodes only.
+
+Another example: In a case where you have different node pools using different availability zones. With a `nodeSelector` using the label `topology.kubernetes.io/zone` you can assign your workload to the nodes in a particular availability zone.
+
+## Adding more node pools
+
+You can add new node pools at any time. You just need to update the cluster configuration and specify the details of the node pools that you want to add.
+
+For example, if this was the cluster configuration
+
+```yaml
+metadata:
+  description: "my cluster"
+  name: test-cluster
+  organization: giantswarm
+nodePools:
+  nodepool0:
+    maxSize: 4
+    minSize: 3
+    instanceTypeOverrides:
+    - r6i.xlarge
+    - r5.xlarge
+    - m5.xlarge
+```
+
+We can add a new node pool like this
+
+```yaml
+metadata:
+  description: "my cluster"
+  name: test-cluster
+  organization: giantswarm
+nodePools:
+  nodepool0:
+    instanceType: r6i.xlarge
+    maxSize: 4
+    minSize: 3
+  nodepool1:
+    instanceType: m5.xlarge
+    maxSize: 3
+    minSize: 1
+```
+
+All the nodes from each node pool will be labeled accordingly.
+
+## Updating an existing node pool
+
+Instances in the node pool will be rolled whenever these properties are changed in the node pool definition:
+- `instanceType`
+- `additionalSecurityGroups`
+
+Instances will also be rolled if these values are changed:
+- `providerSpecific.ami`
+
+### What happens when a node pool is updated
+
+When node pool instances need to be rolled, each instance receives a terminate signal from AWS.
+This is propagated as shutdown signal to the OS and then to each running process like the `kubelet`, which will send a `NodeShutDown` event to the pod.
+
+The `kubelet` will wait up to 5 minutes for all pods to terminate and after that it will terminate itself and the shutdown of the AWS EC2 instance will finally proceed.
+
+Please, be aware that it may happen that AWS decides to force terminate the instance before the 5 minutes.
+
+## Node pool deletion
+
+TBD
```

Comment:
Until this is solved, maybe we can add a tab for vintage already if we support deletion there (I guess since it's a separate CR without Helm "keep" annotation).

Reply:
But we have a different page for `vintage`. The idea is to document vintage on its own page, and use this one only for CAPI.